eng/SharedFramework.External.props

```diff
@@ -9,7 +9,7 @@
   <!-- For the targeting pack, always use packages with PatchVersion=0 -->
   <!-- !!! Need to get ref/ assemblies for these packages some other way in order to feed correct information into Roslyn. !!! -->
   <!-- !!! For example, might update target paths to contain latest implementation assemblies after building /everything/ against the original package content. !!! -->
-  <PropertyGroup Condition="'$(IsServicingBuild)' == 'true' AND ('$(MSBuildProjectName)' == 'Microsoft.AspNetCore.App.Ref' OR '$(IsReferenceAssemblyProject)' == 'true')">
+  <PropertyGroup Condition=" '$(IsServicingBuild)' == 'true' AND '$(MSBuildProjectName)' == 'Microsoft.AspNetCore.App.Ref' ">
```

Comment:
We can probably just move this `PropertyGroup` directly into the ref-pack's project, now that it's only used there

Reply:
It's used everywhere now.